packages/config/config/devices/0x0312/mp22zp.json

```diff
@@ -0,0 +1,80 @@
+{
+	"manufacturer": "Minoston",
+	"manufacturerId": "0x0312",
+	"label": "MP22ZP",
+	"description": "Z-Wave Outdoor Smart Plug with Power Meter",
+	"devices": [
+		{
+			"productType": "0xff00",
+			"productId": "0xff0f",
+			"zwaveAllianceId": 4669
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Basic Set",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"$import": "~/0x0312/templates/minoston_template.json#led_indicator_four_options_inverted"
+		},
+		{
+			"#": "2",
+			"$import": "~/0x0312/templates/minoston_template.json#auto_off_timer"
+		},
+		{
+			"#": "3",
+			"$import": "~/0x0312/templates/minoston_template.json#auto_on_timer"
+		},
+		{
+			"#": "4",
+			"$import": "~/templates/master_template.json#state_after_power_failure_prev_off_on"
+		},
+		{
+			"#": "5",
+			"label": "Power Wattage Reporting Threshold",
+			"valueSize": 1,
+			"unit": "W",
+			"minValue": 5,
+			"maxValue": 50,
+			"defaultValue": 5,
+			"unsigned": true
+		},
+		{
+			"#": "6",
+			"label": "Time Reporting Threshold",
+			"valueSize": 4,
+			"unit": "minutes",
+			"minValue": 1,
+			"maxValue": 65535,
+			"defaultValue": 5,
+			"unsigned": true
+		},
+		{
+			"#": "7",
+			"$import": "~/0x0312/templates/minoston_template.json#power_meter_reporting_threashold_current"
+		},
+		{
+			"#": "8",
+			"$import": "~/0x0312/templates/minoston_template.json#power_meter_reporting_threashold_energy"
+		}
+	],
+	"metadata": {
+		"inclusion": "1. Follow the instructions for your Z-Wave certified controller to add a device to the Z-Wave network.\n2. Once the controller is ready to add your device, press the Manual/Program button on the smart plug 3 time quickly. The blue LED will blink quickly.",
+		"exclusion": "1. Follow the instructions for your Z-Wave certified controller to remove a device to the Z-Wave network.\n2. Once the controller is ready to remove your device, press the manual/program button on the smart plug 3 times quickly.",
+		"reset": "Click the button twice quickly then hold for at least 10 seconds.",
+		"manual": "https://products.z-wavealliance.org/ProductManual/File?folder=&filename=cert_portal/certs/401/marketcert/MP22ZUserManual-1.pdf"
```

Comment:
As with the `zwaveAllianceId`, this manual link seems to be wrong. It's what's published on the Z-Wave Alliance website though. Again, I'll defer to the experts as to whether I should just remove this instead.

Reply:
Replaced with a url from the manufacturer's website which matches the physical document I got in the box.